fuzz/fuzz_targets/spv_parser.rs

```diff
@@ -1,8 +1,10 @@
 #![no_main]
 use libfuzzer_sys::fuzz_target;
 use naga::front::spv::Parser;
```

Comment:
would you mind combining this into `use naga::front::spv::{Options, Parser};`?

Reply:
Hi, @kvark. I have fixed it.